docs/learn-accounts.md

```diff
@@ -100,14 +100,14 @@ Hardware wallet integration from [Ledger](https://ledger.com) is coming soon.
 ## Existential Deposit and Reaping
 
 When you generate an account (address), you only generate a _key_ that lets you access it. The
-account does not exist yet on-chain. For that, it needs the existential deposit - 0.01 KSM / DOT.
-This deposit is paid by the account sending you tokens. The one "creating" you pays for your
-creation.
-
-Going below 0.01 causes an account to be _reaped_. Your account will be wiped from the blockchain's
-state to conserve space. You do not lose access to the reaped address - as long as you have your
-private key or recovery phrase, you can still use the address - but it needs a top-up of another
-existential deposit to be able to interact with the chain.
+account does not exist yet on-chain. For that, it needs the existential deposit - 0.001 KSM (on
```

Comment:
```suggestion
account does not exist yet on-chain. For that, it needs the existential deposit: 0.001 KSM (on
```
Could we change the - to a : instead? It looked like the number was negative

Reply:
```bash
const balances.existentialDeposit: Balance
1.666m KSM
```